generators/maven/index.cjs

```diff
@@ -18,31 +18,41 @@
  */
 /* eslint-disable consistent-return */
 const chalk = require('chalk');
+const { mixBlueprintGenerator } = require('generator-jhipster/support');
 
-const BaseBlueprintGenerator = require('../generator-base-blueprint');
-const { GENERATOR_MAVEN, GENERATOR_PROJECT_NAME, GENERATOR_JAVA_PACKAGE_NAME } = require('../generator-list');
-const { files } = require('./files');
+const { GENERATOR_PROJECT_NAME, GENERATOR_JAVA, GENERATOR_MAVEN } = require('../generator-list');
+const { files } = require('./files.cjs');
+const { MAVEN } = require('./constants.cjs');
+const { BUILD_TOOL } = require('../java/constants.cjs');
 
-module.exports = class extends BaseBlueprintGenerator {
-  constructor(args, opts) {
-    super(args, opts, { unique: 'namespace' });
+const MixedGenerator = mixBlueprintGenerator(GENERATOR_PROJECT_NAME, GENERATOR_JAVA);
```

Comment:
It may be better to get the chain from an external file.

generatorChain.js#getBlueprintSequenceTo(GENERATOR_BUILDTOOL)

Reply:
Done.